### PR TITLE
Update Boxes to 45

### PR DESCRIPTION
**Pull request by A6GibKm**
Needs https://github.com/flathub/org.freedesktop.Sdk.Extension.vala/pull/112

### CODE REVIEW

**Comment by A6GibKm** [comment]
Build fails using the vala-nightly sdk with
```
Cleaning up
Removing files/share/man/man8/intltoolize.8
Removing files/share/man/man8/intltool-update.8
Removing files/share/man/man8/intltool-prepare.8
Removing files/share/man/man8/intltool-merge.8
Removing files/share/man/man8/intltool-extract.8
Removing files/share/man/man8
Removing files/share/man
Removing files/share/intltool/Makefile.in.in
Removing files/share/intltool
Removing files/share/aclocal/intltool.m4
Removing files/share/aclocal
Removing files/share
Removing files/lib/pkgconfig/libusb-1.0.pc
Removing files/lib/pkgconfig
Removing files/lib/libusb-1.0.la
Removing files/include/libusb-1.0/libusb.h
Removing files/include/libusb-1.0
Removing files/include
Removing files/bin/intltoolize
Removing files/bin/intltool-update
Removing files/bin/intltool-prepare
Removing files/bin/intltool-merge
Removing files/bin/intltool-extract
Removing files/bin
Renaming org.gnome.Boxes.appdata.xml to share/appdata/org.gnome.Boxes.appdata.xml
Running appstream-compose
Processing application org.gnome.Boxes
Error loading desktop file: Failed to find icon: Failed to find icon org.gnome.Boxes
Error: ERROR: appstream-compose failed: Child process exited with code 1
```

**Comment by A6GibKm** [comment]
> Build fails using the vala-nightly sdk with
> 
> ```
> Cleaning up
> Removing files/share/man/man8/intltoolize.8
> Removing files/share/man/man8/intltool-update.8
> Removing files/share/man/man8/intltool-prepare.8
> Removing files/share/man/man8/intltool-merge.8
> Removing files/share/man/man8/intltool-extract.8
> Removing files/share/man/man8
> Removing files/share/man
> Removing files/share/intltool/Makefile.in.in
> Removing files/share/intltool
> Removing files/share/aclocal/intltool.m4
> Removing files/share/aclocal
> Removing files/share
> Removing files/lib/pkgconfig/libusb-1.0.pc
> Removing files/lib/pkgconfig
> Removing files/lib/libusb-1.0.la
> Removing files/include/libusb-1.0/libusb.h
> Removing files/include/libusb-1.0
> Removing files/include
> Removing files/bin/intltoolize
> Removing files/bin/intltool-update
> Removing files/bin/intltool-prepare
> Removing files/bin/intltool-merge
> Removing files/bin/intltool-extract
> Removing files/bin
> Renaming org.gnome.Boxes.appdata.xml to share/appdata/org.gnome.Boxes.appdata.xml
> Running appstream-compose
> Processing application org.gnome.Boxes
> Error loading desktop file: Failed to find icon: Failed to find icon org.gnome.Boxes
> Error: ERROR: appstream-compose failed: Child process exited with code 1
> ```

Maybe related to https://github.com/flathub/org.gnome.Boxes/issues/83 ?

**Comment by A6GibKm** [comment]
bot, build

**Comment by A6GibKm** [comment]
Also
```
(/usr/libexec/xdg-desktop-portal:2453015): GLib-GIO-ERROR **: 10:30:40.136: No GSettings schemas are installed on the system
```
I suspect this is due to the gschema file being called org.gnome.boxes.gschema rather than org.gnome.Boxes.gschema

File: org.gnome.Boxes.json
Comment (bbhtt) [inline org.gnome.Boxes.json:1]: Old bug with appstream-glib that makes it fail to find icon when /app/usr exists (ideally it should not exist)

https://github.com/hughsie/appstream-glib/issues/271

Add a global cleanup: `/usr` - contains some udev rules

A few more safe cleanups:

```
/include
/lib/pkgconfig
'*.a'
'*.la'
/share/doc
/share/gtk-doc
/share/man
/share/pkgconfig
```

There's a lot more, but I don't know which ones are actually used so leave them as is.

**Comment by A6GibKm** [comment]
cc @felipeborges

**Comment by felipeborges** [comment]
bot, build

**Pull request merged**